app/src/main/java/org/stepic/droid/core/presenters/CourseJoinerPresenter.kt

```diff
@@ -78,8 +79,9 @@ class CourseJoinerPresenter
         }
         val enrollNotificationClickMillis: Long? = sharedPreferenceHelper.lastClickEnrollNotification
         enrollNotificationClickMillis?.let {
-            val wasClicked = DateTime(it)
-            if (wasClicked.plusMinutes(30).isAfterNow) {
+            val wasClickedPlus30Min = it + 30 * AppConstants.MILLIS_IN_1HOUR
```

Comment:
According to name `wasClickedPlus30Min` I guess there shoulbe be a `AppConstants.MILLIS_IN_1MINUTE` instead.